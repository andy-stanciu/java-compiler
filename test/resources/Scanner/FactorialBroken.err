Unexpected character '_' on line 8 at column 5 of input.
Unexpected character '/' on line 17 at column 5 of input.
