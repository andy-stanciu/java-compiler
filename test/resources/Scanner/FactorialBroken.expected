CLASS ID(Factorial) LCURL PUBLIC STATIC VOID MAIN LPAREN STRING LSQUARE RSQUARE ID(a) RPAREN LCURL PRINT LPAREN NEW ID(Fac) LPAREN RPAREN DOT ID(ComputeFac) LPAREN INT(10) RPAREN RPAREN SEMICOLON RCURL RCURL CLASS ID(Fac) LCURL <UNEXPECTED(_)> ID(test) PUBLIC INT ID(ComputeFac) LPAREN INT ID(num) RPAREN LCURL INT ID(num_aux) SEMICOLON IF LPAREN ID(num) LESS_THAN INT(1) RPAREN ID(num_aux) ASSIGN INT(1) SEMICOLON ELSE ID(num_aux) ASSIGN ID(num) TIMES LPAREN THIS DOT ID(ComputeFac) LPAREN ID(num) MINUS INT(1) RPAREN RPAREN SEMICOLON RETURN ID(num_aux) SEMICOLON RCURL DIV TIMES RCURL 